pytrip/vdx.py

```diff
@@ -267,8 +267,11 @@ def create_dicom(self):
         ds = FileDataset("file", {}, file_meta=meta, preamble=b"\0" * 128)
         if self.cube is not None:
             ds.PatientsName = self.cube.patient_name
+            ds.Manufacturer = self.cube.creation_info  # Manufacturer tag, 0x0008,0x0070 (type LO - Long String)
         else:
-            ds.PatientsName = ""
+            ds.PatientsName = ''
+            ds.Manufacturer = ''  # Manufacturer tag, 0x0008,0x0070 (type LO - Long String)
+        ds.SeriesNumber = '1'  # SeriesNumber tag 0x0020,0x0011 (type IS - Integer String)
         ds.PatientID = "123456"
         ds.PatientsSex = '0'
```

Comment:
PatientsSex: tag (0010,0040); Description: Sex of the named patient. Enumerated Values: M = male F = female O = other.
I think that '0' should be changed to O (other) .